pkgs/os-specific/linux/lvm2/common.nix

```diff
@@ -14,9 +14,6 @@
   udevSupport ? !stdenv.hostPlatform.isStatic,
   udev,
   udevCheckHook,
-  onlyLib ? stdenv.hostPlatform.isStatic,
```

Comment:
Do we want to drop these parameters altogether or just change the defaults?

It has been argued that this is a nixpkgs API change, but I'm unsure if there is any consensus on that and if we should just change that.

Reply:
I removed these parameters because they were only introduced to fix the static compilation, however I am not against restoring them if you think they are useful API. As one of the package maintainers I will leave it to your judgement.